frontend-react/src/pages/Login.tsx

```diff
@@ -1,33 +1,33 @@
-import React from "react";
+import React, { useContext } from "react";
 import { Link, Redirect } from "react-router-dom";
 import { useOktaAuth } from "@okta/okta-react";
 import { SiteAlert } from "@trussworks/react-uswds";
 import { Tokens } from "@okta/okta-auth-js";
 
 import OktaSignInWidget from "../components/OktaSignInWidget";
-import {
-    getOrganizationFromAccessToken,
-    groupToOrg,
-} from "../webreceiver-utils";
-import {
-    setStoredOktaToken,
-    useGlobalContext,
-} from "../components/GlobalContextProvider";
-import { PERMISSIONS } from "../resources/PermissionsResource";
+import { getOrganizationFromAccessToken } from "../webreceiver-utils";
+import { parseOrgs, setStoredOktaToken } from "../contexts/SessionStorageTools";
 import { oktaSignInConfig } from "../oktaConfig";
+import { SessionStorageContext } from "../contexts/SessionStorageContext";
+import { PERMISSIONS } from "../resources/PermissionsResource";
 
 export const Login = () => {
     const { oktaAuth, authState } = useOktaAuth();
-    const { updateOrganization } = useGlobalContext();
+    const { updateSessionStorage } = useContext(SessionStorageContext);
 
     const onSuccess = (tokens: Tokens | undefined) => {
-        let oktaGroups =
-            getOrganizationFromAccessToken(tokens?.accessToken).filter(
-                (group: string) => group !== PERMISSIONS.PRIME_ADMIN
-            ) || [];
+        const parsedOrgs = parseOrgs(
+            getOrganizationFromAccessToken(tokens?.accessToken)
+        );
+        // updateSessionStorage(values)
```

Comment:
Get 'em outta here! So many rogue comments...I'm surprised SonarCloud didn't scream about this one.